katas/content/marking_oracles/palindrome/Solution.qs

```diff
@@ -0,0 +1,14 @@
+namespace Kata {
+    operation Oracle_Palindrome (x : Qubit[], y : Qubit) : Unit is Adj + Ctl {
+        let N = Length(x);
+        // Compute XORs of the bits that should be equal in the first qubits
```

Comment:
We can drop comments from solution code, since it will be covered in the text (I promise!)
```suggestion
```